src/patternfly/components/Pagination/pagination-nav-button.hbs

```diff
@@ -0,0 +1,6 @@
+<button class="pf-c-pagination__nav-button{{#if pagination-nav-button--modifier}} {{pagination-nav-button--modifier}}{{/if}}"
```

Comment:
Was the thinking yesterday to use a button component and its classes but modify for anything particular to pagination?